### PR TITLE
fixed the project list showing the incorrect status and made the proj…

### PR DESCRIPTION
**Pull request by AlexGopal**
…ect badge match the status. Also wrote a cypress test to test this.

### CODE REVIEW

**Comment by mentor-tara** [comment]
You need to be a premium member for this task. Please head to [profy.dev](https://profy.dev#premium) and upgrade your account. Once you're done request another review.

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-11)

**Comment by mentor-tara** [comment]
You need to be a premium member for this task. Please head to [profy.dev](https://profy.dev#premium) and upgrade your account. Once you're done request another review.

File: cypress/e2e/project-list.cy.ts [hunk 1]
```diff
@@ -32,11 +38,45 @@ describe("Project List", () => {
...
+        .find("li")
+        .find(".badge_container__FVLnl")
+        .each(($el, index) => {
+          const status = mockProjects[index].status;
+          const expectedStatus = statusMap[status];
+          const expectedColor = statusColors[expectedStatus];
+
+          console.log("Status:", status);
+          console.log("expectedStatus:", expectedStatus);
+          console.log("expectedColor:", expectedColor);
```
Comment (jkettmann) [inline cypress/e2e/project-list.cy.ts:74]: I'd remove these

File: features/projects/components/project-card/project-card.tsx [hunk 2]
```diff
@@ -16,15 +16,22 @@ const languageNames = {
...
+const statusMap: { [key: string]: ProjectStatus } = {
+  error: ProjectStatus.critical,
+  warning: ProjectStatus.warning,
+  info: ProjectStatus.stable,
+};
+
 export function ProjectCard({ project }: ProjectCardProps) {
   const { name, language, numIssues, numEvents24h, status } = project;
 
+  console.log(status);
```
Comment (jkettmann) [inline features/projects/components/project-card/project-card.tsx:34]: I'd remove this but apart from that great job. This is a clean solution

**Pull request merged**